cmd/flow-dps-server/main.go

```diff
@@ -44,11 +45,13 @@ func main() {
 		flagLevel string
 		flagIndex string
 		flagPort  uint16
+		flagFirst uint64
 	)
 
 	pflag.StringVarP(&flagIndex, "index", "i", "index", "database directory for state index")
 	pflag.StringVarP(&flagLevel, "level", "l", "info", "log output level")
 	pflag.Uint16VarP(&flagPort, "port", "p", 5005, "port to serve GRPC API on")
+	pflag.Uint64VarP(&flagFirst, "first", "f", 0, "first height to fix DB with") // FIXME: remove this after all fixed
```

Comment:
Should be removed according to `FIXME`

Reply:
Yes, but after I fixed the `mainnet-7` index.